upstream-community-operators/crossplane/0.9.0/crossplane.v0.9.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,113 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: crossplane.v0.9.0
+  namespace: placeholder
+  annotations:
+    capabilities: "Basic Install"
+    categories: "OpenShift Optional"
+    containerImage: "crossplane/crossplane:v0.9.0"
+    createdAt: 2020-04-01T11:59:59Z
+    support: Crossplane Developers
+    repository: https://github.com/crossplane/crossplane
+    description: |-
+      Crossplane is an open source control plane that allows you to manage
+      applications and infrastructure the Kubernetes way. It manages external
+      services by installing Custom Resource Definitions
+      (CRDs) and reconciling instances of those Custom Resources. Crossplane is
+      built to be extensible, meaning that anyone can add functionality for an new or existing cloud provider.
+    alm-examples: |-
+      []
```

Comment:
I'll provide an example for each of the 22 included CRDs, unless that will overwhelm the UI.

Reply:
it shouldn't overwhelm the UI